packages/paste-core/components/blockquote/src/BlockquoteSource.tsx

```diff
@@ -0,0 +1,64 @@
+import { Anchor } from "@twilio-paste/anchor";
+import { Box, type BoxProps } from "@twilio-paste/box";
+import { Text } from "@twilio-paste/text";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import React from "react";
+
+import { BlockquoteContext } from "./BlockquoteContext";
+
+export interface BlockquoteSourceProps extends HTMLPasteProps<"div"> {
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   * @default 'BLOCKQUOTE_SOURCE'
+   * @type {BoxProps['element']}
```

Comment:
```suggestion
   * @type string
```

right? or am I doing it wrong.

Reply:
it always ends up being a string, but we use the box props type just to make sure all types inherit any future changes